### PR TITLE
Altair Add back minimal spec tests

### PR DESCRIPTION
**Pull request by terencechain**
Altair's minimal spec tests were ignored. This PR fixed it by using the correct ssz sizes with bazel

### CODE REVIEW

File: proto/eth/v1alpha1/BUILD.bazel [hunk 1]
```diff
@@ -153,6 +154,18 @@ go_library(
...
+    name = "ssz_proto_files",
+    srcs = [
+        "beacon_block_altair.proto",
+    ],
+    config = select({
+        "//conditions:default": "mainnet",
+        "//proto:ssz_mainnet": "mainnet",
+        "//proto:ssz_minimal": "minimal",
+    }),
+)
```
Comment (prestonvanloon) [inline proto/eth/v1alpha1/BUILD.bazel:167]: LGTM

File: proto/eth/v1alpha1/beacon_block_altair.proto [hunk 2]
```diff
@@ -92,7 +92,7 @@ message BeaconBlockBodyAltair {
 // support light client infra.
 message SyncAggregate {
   // Sync committee bits as Bitvector to track votes.
-  bytes sync_committee_bits = 1 [(ethereum.eth.ext.ssz_size) = "64", (ethereum.eth.ext.cast_type) = "github.com/prysmaticlabs/go-bitfield.Bitvector512"];
+  bytes sync_committee_bits = 1 [(ethereum.eth.ext.ssz_size) = "sync_committee_bytes.size", (ethereum.eth.ext.cast_type) = "sync_committee_bits.type"];
```
Comment (prestonvanloon) [inline proto/eth/v1alpha1/beacon_block_altair.proto:95]: Really nice naming here. Very straight forward with the substitutions.

**Review by prestonvanloon — APPROVED** [review]

**Review by prestonvanloon — APPROVED** [review]
LGTM

**Comment by terencechain** [comment]
Ran locally, for minimal spec tests, memory was about ~3.7GB. Not sure if that's high enough to cause buildkite agent to go oom

**Comment by terencechain** [comment]
Merging this first, we can fix oom in a separate issue/thread as it happens in all the altair PRs

**Pull request merged**